code/drasil-lang/lib/Drasil/Language/Concept.hs

```diff
@@ -1,29 +1,53 @@
 {-# Language TemplateHaskell #-}
 -- | Define concept-related chunks. A concept is usually something that has
 -- a term, definition, and comes from some domain of knowledge.
-module Language.Drasil.Chunk.Concept.Core(
-  -- * Concept-related Datatypes
-  ConceptChunk(ConDict)
-  , ConceptInstance(ConInst)
-  , sDom)
+module Drasil.Language.Concept (
+    ConceptChunk(ConDict), dcc, dccWDS, cc, cc', ccs, cw
+  , ConceptInstance(ConInst), cic
+  , sDom
+  , Definition(..), ConceptDomain(..), Concept
+  )
   where
-import Language.Drasil.ShortName (HasShortName(..), ShortName)
-import Language.Drasil.Classes (NamedIdea(term), Idea(getA),
-  Definition(defn), ConceptDomain(cdom))
-import Language.Drasil.Chunk.NamedIdea (IdeaDict)
-import Language.Drasil.Label.Type ((+::+), defer, name, raw,
-  LblType(..), Referable(..), HasRefAddress(..))
-import Language.Drasil.Sentence (Sentence)
+
+import Control.Lens (makeLenses, (^.), view, Lens')
+
 import Drasil.Database.UID (UID, HasUID(..))
+import Drasil.Language.Idea (IdeaDict, NamedIdea(..), Idea(..), nw, nc, mkIdea)
+import Drasil.Language.ShortName (HasShortName(..), ShortName, shortname')
 
-import Control.Lens (makeLenses, (^.), view)
+import Language.Drasil.NounPhrase (NP, pn)
+import Language.Drasil.Label.Type ((+::+), defer, name, raw,
+  LblType(..), Referable(..), HasRefAddress(..))
+import Language.Drasil.Sentence (Sentence(S))
 
 -- | Check if something has one domain. Throws an error if there is more than one.
-sDom :: [UID] -> UID
+sDom :: [UID] -> UID -- FIXME: Why is this useful? It doesn't look like it should be!
 sDom [d] = d
 sDom d = error $ "Expected ConceptDomain to have a single domain, found " ++
   show (length d) ++ " instead."
 
+-- TODO: conceptual typeclass?
+-- TODO: I was thinking of splitting QDefinitions into Definitions with 2 type variables
```

Comment:
`QDefinition`, while an awful name, was still better: a `QDefinition` grabbed the definition of a *quantity*. That quantities must have a definition makes sense. `Definition` on its own, not so much.